jupyter-extension/jupyterlab_pachyderm/handlers.py

```diff
@@ -664,11 +665,11 @@ def setup_handlers(web_app, config_file: Path):
             f"Created Pachyderm client for {client.address} from local config"
         )
     except FileNotFoundError:
-        if PACHD_ADDRESS:
```

Comment:
unrelated to this PR, but just clarifying. We want to use pachd_address/dex_token passed through env vars only if no config file is present? And then it doesn't appear that this is written to a config file - should it be?

Reply:
that is correct. good call on this not writing to config file, i think it makes sense to write it. i have added that logic and added testing for it.